sources.md

```diff
@@ -0,0 +1,82 @@
+# Data Sources
+
+This project uses data from various sources that are openly licensed or in the public domain. Below are the sources and their respective information:
+
+## CC Legal Tools
+
+**Description:** A .txt provided by Timid Robot containing all legal tool paths. The data from Google Custom Search will only cover 50+ general, most significant categories of CC License for data collection quota constraint. As an additional note, the order of precedence of license the collected data's first column is sorted due to intermediate data analysis progress.
+
+**API documentation link:** [List of all current CC legal tool paths by TimidRobot](https://gist.github.com/TimidRobot/6c7f9a1b0d8e5a3c4f2a6b0d5e5f4c7b)
+
+**API information:**
+- No API key required
+- No query limits
+
+## Flickr
+
+**Description:** The Flickr API exposes identifiers for users, photos, photosets and other uniquely identifiable objects.
```

Comment:
This statement provides an abstract view of the Flickr API functionality. Perhaps something more concrete would be better like - 
The Flickr API enables developers to access and manipulate photo and media content from the Flickr platform through a set of web services. @TimidRobot kindly provide suggestion.

Reply:
Thank you for your suggestion. While I appreciate your input, I believe that my original statement provides a clear and concise overview of the Flickr API’s functionality. However, I am open to feedback and will consider incorporating more specific details in the future. @TimidRobot, do you have any thoughts on this matter?